Tests/TestingTests/SourceLocationTests.swift

```diff
@@ -51,21 +51,6 @@ struct SourceLocationTests {
     #expect(sourceLocation.fileName == "D.swift")
   }
 
-
-#if !SWT_NO_EXIT_TESTS
```

Comment:
I just moved this down without modification, to consolidate several exit tests together